templates/monitoring/kube_state_metrics_3scale_alerts.yaml

```diff
@@ -94,15 +94,24 @@ spec:
         for: 5m
         labels:
           severity: critical
-      - alert: ThreeScalePodCount
+      - alert: ThreeScaleZyncPodAvailability
         annotations:
+          message:  3Scale Zync has {{ "{{" }} printf "%.0f" $value {{ "}}" }} pods in a ready state. Expected at minimum of 2 pods.
```

Comment:
One small thing - `Expected at minimum of 2 pods`. I would reword this to: `Expected a minimum of 2 pods`. Same for the other alert.

Reply:
thanks @obrienrobert  :+1: